File: Modules/HumanResource/PayrollSettings/PayrollSettings/TestingMode.cs
using System.ComponentModel;
namespace HRMSoftware.PayrollSettings;
[Serenity.ComponentModel.EnumKey("HumanResource.PayrollSettings.TestingMode")]
public enum TestingMode
{

    [Description("No")]
    No = 0,
    [Description("Yes")]
    Yes = 1
}

File: Modules/ServerTypes/PayrollSettings/TestingMode.ts
import { Decorators } from "@serenity-is/corelib";

export enum TestingMode {
    No = 0,
    Yes = 1
}
Decorators.registerEnumType(TestingMode, 'HRMSoftware.PayrollSettings.TestingMode', 'HumanResource.PayrollSettings.TestingMode');